pkg/builder/builder.go

```diff
@@ -1467,12 +1467,18 @@ func buildNMACommand() []string {
 }
 
 // buildScrutinizeCmd returns the full vcluster scrutinize command
-func buildScrutinizeCmd(args []string) []string {
+func buildScrutinizeCmd(args []string, vdb *vapi.VerticaDB) []string {
 	cmd := []string{
 		"/opt/vertica/bin/vcluster",
 		"scrutinize",
 	}
 	cmd = append(cmd, args...)
+	// if there is no password, we need to explicitly
+	// set the password flag with empty string as value,
+	// to still assume password as the authentication method
+	if vdb.Spec.PasswordSecret == "" {
+		cmd = append(cmd, "--password=")
```

Comment:
I think we should move this to buildScrutinizeCmdArgs. It makes more sense to keep the arg building all in one place.